devenv/setenv.sh

```diff
@@ -0,0 +1,19 @@
+export B2ACCESS_CONSUMER_KEY="<B2ACCESS_USERNAME_FOR_OAUTH2_CLIENT>"
+export B2ACCESS_SECRET_KEY="<B2ACCESS_PASSWORD_FOR_OAUTH2_CLIENT>"
+export B2SHARE_ACCOUNTS_SESSION_REDIS_URL="redis://localhost:6379/0"
+export B2SHARE_BROKER_URL="redis://localhost:6379/0"
+export B2SHARE_CELERY_RESULT_BACKEND="redis://localhost:6379/1"
+export B2SHARE_JSONSCHEMAS_HOST="localhost:5000"
+export B2SHARE_SEARCH_ELASTIC_HOSTS="localhost:9200"
+export B2SHARE_SERVER_NAME="localhost:5000"
+export B2SHARE_UI_PATH="/Users/thijsc/.pyenv/versions/b2share-evolution/src/webui/app"
+export B2SHARE_SECRET_KEY=mfTpGpQBwTTOmWCoZrYSuiJ1UzXq1d2L
+export FLASK_DEBUG=0
+export USE_STAGING_B2ACCESS=1
+export B2SHARE_DATADIR=<path-to>/b2share-datadir
+export B2SHARE_POSTGRESQL_PASSWORD=xxxxxxx
```

Comment:
same here. Better put a temporary password than use something the users need to change.